packages/app/obojobo-express/http_server.js

```diff
@@ -25,12 +23,14 @@ const normalizePort = val => {
 	return false
 }
 
-const startServer = (app, logger, port = '3000') => {
+const startServer = (app, logger, port = '3000', options = {}) => {
 	console.log('Note: Logging and config options can be set using environment variables.')
+	console.log(`DEBUG is set to: ${process.env.DEBUG}`)
 
 	port = normalizePort(port)
 	app.set('port', port)
-	const server = http.createServer(app)
+	const http_or_https = options.cert ? require('https') : require('http')
```

Comment:
Should we throw an error here if they pass in the 'cert' option but omit the 'key' option, or just let it ride?

Reply:
I'm fine with relying on `https` to complain about missing something.